runtime-jvm/main/src/main/scala/util/Bytes.scala

```diff
@@ -115,8 +115,8 @@ object Bytes {
 
     def emptyBase: Base = One(Array())
 
-    case object OutOfBounds extends Throwable
-    case object NotFound extends Throwable
+    case class OutOfBounds() extends Throwable
+    case class NotFound() extends Throwable
```

Comment:
I've made these classes rather than objects to make stack traces useful. A `Throwable` will forever retain the stack trace from the first place it was thrown. Exception objects don't play nice with stack traces in general since stack tracing is a side effect.

Reply:
That is interesting.

Reply:
That is okay for debugging but we should switch back to `object` once we've gotten all the kinks out - otherwise we are paying a performance cost that exceptions were meant to avoid (vs using `Option`).